resources/zib/zib-MultidisciplinaryTeamMeeting.xml

```diff
@@ -0,0 +1,239 @@
+      <mapping>
+        <identity value="zib-multidisciplinaryteammeeting-v1.0-2020EN" />
+        <map value="NL-CM:15.1.3" />
+        <comment value="PatientConsultationDateTime (implicit, main mapping is on `Encounter.period`)" />
+      </mapping>
+    </element>
+    <element id="Encounter.type">
+      <path value="Encounter.type" />
+      <slicing>
+        <discriminator>
+          <type value="pattern" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="1" />
+    </element>
+    <element id="Encounter.type:multidisciplinaryTeamMeetingCode">
+      <path value="Encounter.type" />
+      <sliceName value="multidisciplinaryTeamMeetingCode" />
+      <min value="1" />
+      <max value="1" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="384682003" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Encounter.subject">
+      <path value="Encounter.subject" />
+      <short value="The patient or group related to this encounter" />
+      <definition value="The patient or group related to this encounter. (Note: this definition is a pre-adopt of https://jira.hl7.org/browse/FHIR-20479. Referencing the Patient as `.subject` does not indicate its presence at the Encounter. Use `.participant.individual.extension:patient` for this.)" />
+    </element>
+    <element id="Encounter.participant">
+      <path value="Encounter.participant" />
+      <short value="Participant" />
+      <alias value="Deelnemer" />
+      <mapping>
+        <identity value="zib-multidisciplinaryteammeeting-v1.0-2020EN" />
+        <map value="NL-CM:15.2.13" />
+        <comment value="Participant" />
+      </mapping>
+    </element>
+    <element id="Encounter.participant.type">
+      <path value="Encounter.participant.type" />
+      <short value="RolDeelnemer" />
+      <definition value="Role of the person in the patient meeting, for example chairman, expert, notifier, secretary." />
+      <alias value="RolDeelnemer" />
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.15.2.2--20200901000000" />
```

Comment:
I think it would be better to slice the 'participant.type' before adding a ValueSet binding. This because you are allowed to add multiple types and in this way you are constraining all of them, while I think just one of them should conform to this ValueSet.

Reply:
Looking at it I think it is actually best to slice `participant` as a whole, which then matches the Particpant container of the zib. Either way is better than it is now actually

Reply:
Hi @LilianMinne,
When looking at this again I notice the following things:

- Since the binding of the ValueSet is extensible, any slice involving type as a discriminator does not have any validation power, so it seems hard to change anything in a meaningful way.
- The cardinality of RolDeelnemer in the zib is actually 0..* so it seems the binding is _meant_ to be constraining all of them following the zib.
- The ValueSet includes the code 'OTH'.

The 'extensible' binding and the OTH code actually give two options to 'bypass' the ValueSet. Is there any other use case that you can think of that you cannot apply to this profile? If so, I think we need to pass it to the zib-centrum but we can always 'pre-adopt' if they acknowledge it.

Reply:
Hi @jduwel what I meant is that (although you could add multiple types conforming to the RolDeelnemer in the zib) there may be cases in which implementers would like to send additional type information about the participant. 

I cannot really think of a use case right now, but we often see in hospitals implementing the FHIR profiles that beside the zib values they want to be able to send additional codes. As long as it is a code from a different terminology system describing the same concept they may of course add an additional coding. But in some cases they wish to add an entirely different valueset describing a different concept. 
For example, in EpisodeOfCare.diagnosis.role they want to add 2 roles: one saying this is for example a primary or secondary diagnosis and one saying for example this is an admission or discharge diagnosis. (this is actually a bad example since the max cardinality of this element is 1, but you get the point)

The extensible binding indeed provides a way out in most cases, unless there is overlap in the codes. But it is bit confusing to combine codes from different valuesets describing different concepts. I agree however that slicing won't exactly solve this problem as it will be hard to add a meaningful discriminator.

So, although my feeling says that based on open world modeling principles we should only constrain the valueset at slice level, it probably makes no sense to do so if there's no technical difference.